pages/about.tsx

```diff
@@ -1,11 +1,42 @@
 import { VStack } from '@chakra-ui/react'
+import { InferGetStaticPropsType } from 'next'
+import { RichText } from '@graphcms/rich-text-react-renderer'
 
 import { Heading } from 'components/Base/Heading'
-import { HighlightLink } from 'components/Base/HighlightLink'
+import { ImageWithCaptions } from 'components/ImageWithCaptions'
+import { getAboutMePage } from 'graphql/queries/getAboutMePage'
 import { Paragraph } from 'components/Base/Paragraph'
-import { PlaceImage } from 'components/PlaceImage'
+import { HighlightLink } from 'components/Base/HighlightLink'
+
+export const getStaticProps = async () => {
+  try {
+    const { sections } = await getAboutMePage()
+
+    return {
+      props: {
+        sections
+      }
+    }
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error)
+
+    return {
+      props: {}
+    }
+  }
+}
 
-export default function About () {
+const RichTextParagraph = ({ children }) => <Paragraph variant='regular'>{children}</Paragraph>
+const RichTextAnchor = ({ children, href }) => <HighlightLink href={href}>{children}</HighlightLink>
```

Comment:
Extracted to separate variables, instead of passing those functions as simple `richTextCustomElements` object properties, in order to expose the `displayName` to the React tree